topics/ecology/tutorials/lotus2-identifying-fungi/tutorial.md

```diff
@@ -0,0 +1,462 @@
+> >
+> > 1. Total reads in matrix: 132123
+> > 2. Phylum	95%, Genus	65%
+> > 3. 00:02:34 (i.e. 2 minutes 34 seconds)
+> > 4. In the *"OTU abundance matrix"*, the OTUs are arranged in order of greatest-to-least abundance so OTU1 was the most abundant with 0 reads in sample **C** (Control), 25,659 reads in sample **N5**, and 0 reads in sample **Pcov3**
+> > 5. In the *"OTU abundance matrix"*, the two least abundant sequences with 6 reads total each were OTU256 and OTU257. We can look up the sequences of these files in the *"FASTA-formatted extended OTU seed sequences"* at the bottom of the file:
+> >
+> > ```
+> > >OTU256
+> > GAAATGCGATACGTAATGCGAATTGCAGAACTCAGTGAATCATCAAATCTTTGAACGCAAATTGCGCTTTCGGGATAGGCCTGAAAGCACGTTTCTTTGAGTATCGATTCAACCAACGCTCGATTGCGTGCTTTTTATTTTTCAAAATAAAAGCTTGCATTCGGTTGTAATGAGTTTTCTTTCTCTTTGAAAGTGACTTGAAGAATCGACAGTGAATGAACGATTTTCAAATCGAAACATCTGTCGGCATAAGCAATGCGGTTAAACTATTGCGTTGTGAGCTGATAGGATGTAGGCGATGTAAGAAATCGTTGGATCTTGTAACTGTTCGCAAGTGACAAGAATGACAAAATTTGATTAAGATCTCAAATGAAGCGAGGATACCCGCCGAACTTAA
+> > >OTU257
+> > GAAATGCGATAAGTAGTGTGAATTGCAGAATTCAGTGAATCATCGAATCTTTGAACGCAAATGGCGCTCCCTGGCATCCCGGGAAGCATGCCTGTTTGAGAACCGTAGATAGGCCAAGCTCACCAGGTTCCCCGTATCATCATCCAGCCCGGACGATGTTTACCACTCTGGTGGTGATGGGTCACTGGCGCTTTTTCGCTGCCCGTGACCTGCAACCTTTTGTCCCCGGAAGTCGACGGACCCGGATTCCCATGTCAGGTTGACACGGAGAGGGCATCACCCCCCAATCCGATGGTTTTCTGGGGAAAGACGCGGCCATCCTTCCCGGGGCGCCCCTTCAACaaaacaaaaaaaaacaaaaaaaaaCTTGGTCTCAAATCAGGCAAGAGAACCCGCTGAACTTAA
+> > ```
+> >
+> {: .solution}
+>
+{: .question}
+
+
+# Run LotuS2 on your own data
+
+To run the LotuS2 workflow on your own fungal data, you only need the sequencing fastq files and some information about what primers were used.
+
+In the example dataset, we used the ITS3/ITS4 primer pair to sequence the ITS2 reqion, so our Forward primer was `GCATCGATGAAGAACGCAGC` and the Reverse primer was `TCCTCCGCTTATTGATATGC`
+
+When you want to process your own sequencing files, you can specify the Forward and Reverse primer in the Galaxy LotuS2 Parameters directly in the options if all your samples have the same forward and reverse primers:
+  - *"Forward primer used to amplify DNA region (optional) - optional"*: `GCATCGATGAAGAACGCAGC`
+  - *"Reverse primer used to amplify DNA region (optional) - optional"*: `TCCTCCGCTTATTGATATGC`
+
+We advise adding metadata to the phyloseq object after running LotuS2, so that a large mapping file with all metadata does not have to be created. See future tutorials on adding metadata from a .csv file to a phyloseq object in R.
+
+## Create your own mapping tsv file
+
+Mapping files for LotuS2 have to be in tab-separated-values format. You can create this file in any spreadsheet sofware, eg: Microsoft Excel or Google Sheets. The important thing to note is that when you save or export the file, you should select the TSV format (called "Tab separated values" in Google Sheets or "Tab-delimited text .txt" in Excel)
```

Comment:
if you already got it in csv it can be transformed in galaxy with https://usegalaxy.eu/?tool_id=Convert%20characters1&version=latest